data-manipulation/encryption/camellia/encrypt-data-using-camellia.yml

```diff
@@ -33,3 +35,16 @@ rule:
           - bytes: 1c 6f d3 f1 a5 53 ff 54 = sigma4
           - bytes: 1d 2d 68 de fa 27 e5 10 = sigma5
           - bytes: fd c1 e6 b3 c2 88 56 b0 = sigma6
+        - and:
+          - string: /A09E667F3BCC908B/i
+            description: sigma1_str
+          - string: /B67AE8584CAA73B
+            description: sigma2_str
+          - string: /C6EF372FE94F82BE/i
+            description: sigma3_str
+          - string: /54FF53A5F1D36F1C/i
+            description: sigma4_str
+          - string: /10E527FADE682D1D/i
+            description: sigma5_str
+          - string: /B05688C2B3E6C1FD/i
+            description: sigma6_str
```

Comment:
i can't believe these are actually included as ASCII strings, but you're certainly right!

![image](https://user-images.githubusercontent.com/156560/91611194-f8828d80-e937-11ea-8652-9ece88f56b61.png)